app/views/reports/regions/cohort.html.erb

```diff
@@ -42,7 +42,7 @@
                     data-toggle="tooltip"
                     data-placement="top"
                     data-trigger="hover focus click"
-                    title="<%= number_with_delimiter(cohort["no_bp"]) %> patients didn't have a BP taken"
+                    title="<%= number_with_delimiter(cohort["no_bp"]) %> <% if cohort["no_bp"] == 1 %>patient<% else %>patients<% end %> didn't have a BP taken"
```

Comment:
[pluralize](https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize) is your friend here (and below) .  No need to do this by hand. 😁

Reply:
Thanks Rails :heart:

Reply:
Pluralize is so nice ❤️